pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "cps"
+version = "0.dev"
+description = ""
+authors = ["The CPS Project <@>"]
```

Comment:
The `authors` field is a requirement or poetry won't run. We can always change the name. I honestly just put it there as a placeholder. (Hence the `<@>`, as poetry just checks that there is an `@` to make sure it's a "valid" email address)

Reply:
In that case, I suppose we should at least use the mailing list address instead of (null)@(null). :wink:

Reply:
For what it's worth, this field is only valuable if you're publishing to pypi. Otherwise it *does* sort of run the risk of getting scraped and added to a spam list 😮‍💨

Reply:
:shrug: I'm okay with it, then. @bretbrownjr?

Reply:
Ah, upon rereading Brett's original comment, IIUC he's also okay with this.